app/services/search_service.rb

```diff
@@ -94,7 +94,7 @@ def full_text_searchable?
   end
 
   def account_searchable?
-    account_search? && !(@query.include?('@') && @query.include?(' '))
+    account_search? && !(@query.start_with?('#') || (@query.include?('@') && @query.include?(' ')))
```

Comment:
Actually since hashtags can't contain a space either the paranthesis should be around the or and not the and.

Reply:
:thinking:

On the contrary,
```ruby
account_search? && !((@query.start_with?('#') || @query.include?('@')) && @query.include?(' '))
```
would allow '#foo' to return accounts but not '#foo bar', which isn't what we want